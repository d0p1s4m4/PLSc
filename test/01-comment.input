/* this is a comment */

/* also a
a comment
but using
multiple line */






/* comment */
