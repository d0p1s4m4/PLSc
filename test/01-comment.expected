{
	"file": "test/01-comment.input",
	"tokens": [],
	"errors": []
}